logback/logback/build.gradle

```diff
@@ -0,0 +1,22 @@
+dependencies {
+    if (project.ext.targetJavaVersion >= 11) {
```

Comment:
Q: Could you simply add `:logback12` as `api` dependency instead of adding `logback12Dir` resources?
```gradle
api project(":logback12")
```

Reply:
I haven't tried yet, but I would guess `testImplementation` won't bring in transitive dependencies.  Let me try though 😄

Reply:
<img width="888" alt="Screenshot 2023-08-16 at 11 45 41 AM" src="https://github.com/line/armeria/assets/8510579/4f4d05e5-a90e-401a-bee5-c52f0396f67e">

Got the following jar when doing so, let me know if you have a better idea 😅 

```
dependencies {
    // Logback
    api project(':logback12')
}
```

Reply:
Right, I imagined `armeria-logback.jar` could be (almost) empty and `armeria-logback.xml` has a dependency for `armeria-logback12.jar`